tests/scipy_optimize_test.py

```diff
@@ -231,6 +231,59 @@ def min_op(z0):
     jax_res = min_op(init)
     self.assertAllClose(jax_res, expect, atol=2e-5)
 
+class TestLSA(jtu.JaxTestCase):
+  """
+  Tests for linear_sum_assignment.
+  """
+
+  @jtu.sample_product(
+    maximize=[False, True],
+    shape=[(10, 10), (100, 60), (60, 100)],
+    generator=[
+        lambda shape: np.random.uniform(0, 1, shape),
+        lambda shape : np.random.randint(0, 10, shape),
```

Comment:
Please avoid using irreproducible random seeds in tests: it may lead to flaky test failures that are hard to reproduce. Take a look at how other JAX tests are implemented (e.g. here you could use `jtu.rand_default` with `dtype='int32'` and `dtype='float32'`)